### ISSUE TITLE
Feature Request: Persistent Trigger

### ISSUE THREAD

**Issue by FrenchToucan** [markusfisch/PieLauncher#30]
It would be great to have a persistent trigger on the edge of the screen that could trigger the pie menu regardless nof what is on screen.

**Comment by markusfisch** [comment]
Yes, had the very same idea already 👍 I hope to add this feature soon.
